prov/netdir/netdir.h

```diff
@@ -0,0 +1,190 @@
+/*
+* Copyright (c) 2015-2016 Intel Corporation, Inc.  All rights reserved.
+*
+* This software is available to you under a choice of one of two
+* licenses.  You may choose to be licensed under the terms of the GNU
+* General Public License (GPL) Version 2, available from the file
+* COPYING in the main directory of this source tree, or the
+* BSD license below:
+*
+*     Redistribution and use in source and binary forms, with or
+*     without modification, are permitted provided that the following
+*     conditions are met:
+*
+*      - Redistributions of source code must retain the above
+*        copyright notice, this list of conditions and the following
+*        disclaimer.
+*
+*      - Redistributions in binary form must reproduce the above
+*        copyright notice, this list of conditions and the following
+*        disclaimer in the documentation and/or other materials
+*        provided with the distribution.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+* NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+* BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+* ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+* CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*/
+
+#ifndef _FI_NETDIR_H_
+#define _FI_NETDIR_H_
+
+#if HAVE_CONFIG_H
+#  include <config.h>
+#endif /* HAVE_CONFIG_H */
+
+#include <ws2tcpip.h>
+#include <winsock2.h>
+
+#include <ndspi.h>
+#include "fi_osd.h"
+
+#include "netdir_assert.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+#define OFI_ND_MAJOR_VERSION 0
+#define OFI_ND_MINOR_VERSION 1
+
+#define ND_MSG_IOV_LIMIT (256)
+#define ND_MSG_INTERNAL_IOV_LIMIT (512)
+
+#define ND_EP_MAX_CM_DATA_SIZE (256)
+
+#define OFI_ND_PRIMARY_CAPS					\
+	FI_MSG | FI_RMA | FI_SEND | FI_RECV |			\
+	FI_READ | FI_WRITE | FI_REMOTE_READ | FI_REMOTE_WRITE
+#define OFI_ND_CAPS OFI_ND_PRIMARY_CAPS
+
+#define ofi_nd_zeromem(ptr) ZeroMemory((ptr), sizeof(*(ptr)))
```

Comment:
Do you mean ofi_nd_zeromem is unnecessary? It's just for simplification. Or does your comment about OFI_ND_PRIMARY_CAPS?